WalletWasabi.Gui/Controls/WalletExplorer/CoinListViewModel.cs

```diff
@@ -23,6 +23,14 @@
 
 namespace WalletWasabi.Gui.Controls.WalletExplorer
 {
+	internal enum CoinListViewSortTarget
```

Comment:
If you use my nameof idea and save the name of the SortOrder property as a string you don't need this.